deploy/cluster-monitoring-config/config.yaml

```
@@ -0,0 +1,6 @@
+deploymentMode: "SelectorSyncSet"
+selectorSyncSet:
+  matchExpressions:
+    - key: hive.openshift.io/version-major-minor
+      operator: In
+      values: ["4.6"]
```

```suggestion
      values: ["4.6","4.7,"4.8"]
```

Just so we don't have to come back to this later.  At 4.8+ policy will be we don't support versions of OCP on OSD that don't support UWM and this can condition be removed.